circle.yml

```diff
@@ -0,0 +1,61 @@
+---
+version: 2
+
+buildSteps: &buildSteps
+  - checkout
+  - restore_cache:
+      keys:
+        - dependency-cache-{{ checksum "package.json" }}
```

Comment:
I don't see you setting this key anywhere. Am I missing something about how circle caching works?

Reply:
You're right, I'd mixed up two examples and ended up with mismatched keys. This is supposed to be set in the `save_cache` step below. Now fixed (I think).

Reply:
Yep, tested, it seems to now be happily picking up this cache: https://circleci.com/gh/resin-io-modules/resin-pine/34